apps/search/views.py

```diff
@@ -35,3 +38,36 @@ def search(request):
             "search_results": search_results,
         },
     )
+
+
+class PageSerializer(serializers.ModelSerializer):
+    parent_section = serializers.SerializerMethodField("get_parent_section")
+
+    class Meta:
+        model = Page
+        fields = ["id", "title", "search_description", "url_path", "parent_section"]
+
+    def get_parent_section(self, page):
+        ancestors = page.get_ancestors()
+        if len(ancestors) >= 3:
+            return ancestors[2].title
+        else:
+            return "Home"
+
+
+@api_view(["GET"])
+def search_json(request):
+
```

Comment:
Remove the empty line?

Reply:
I always run `make format`. Shouldn't that take care of this?